web/src/selectors/Assertion.selectors.ts

```diff
@@ -17,9 +17,46 @@ const selectAffectedSpanList = createSelector(stateSelector, paramsSelector, (st
   return trace?.spans.filter(({id}) => spanIdList.includes(id)) || [];
 });
 
+const selectResultAssertionsBySpan = createSelector(
```

Comment:
New selector that transforms the `run` data into a hashed object by attribute (filtered by span), something like this:
```js
{
    "cloud.availability_zone": {
        "failed": [
            {
                "id": "span[tracetest.span.type = \"database\"  service.name = \"pokeshop\"]",
                "label": "database pokeshop "
            },
            {
                "id": "span[tracetest.span.type = \"database\"]",
                "label": "database "
            }
        ],
        "passed": [
            {
                "id": "span[name = \"pg.query:SELECT\"  tracetest.span.type = \"database\"  service.name = \"pokeshop\"  db.system = \"postgresql\"  db.name = \"pokeshop\"  db.user = \"ashketchum\"]",
                "label": "pg.query:SELECT database pokeshop postgresql pokeshop ashketchum "
            }
        ]
    }
}
```